request.go

```diff
@@ -61,12 +61,12 @@ type PathConf struct {
 }
 
 type QueryConf struct {
-	// true - "?id=1&id=2&id=3", false - "?id=1,2,3"
-	Exploded bool
 	// one of QueryStyleForm, QueryStyleSpace, QueryStylePipe or QueryStyleDeep
 	Style string
 	// one of QueryDelimiterPipe, QueryDelimiterSpace, QueryDelimiterComma
 	Delimiter string
+	// true - "?id=1&id=2&id=3", false - "?id=1,2,3"
+	Exploded bool
```

Comment:
In the example you have "imploded" flag, here it's the opposite. Should stick to one of them.

Reply:
In the field tag, you can use both - `imploded` and `exploded`. Zero value of the `Exploded` field represents the implicit state.